### ISSUE TITLE
CSS Styles prefix

### ISSUE THREAD

**Issue by Avejack** [stephanrauh/ngx-extended-pdf-viewer#245]
**Is your feature request related to a problem? Please describe.**
This library overrides my scss style class definitions. For example, I am constantly using the class-name "body" but your library sets styles to this class.

**Describe the solution you'd like**
It would be better if all classes would get a prefix like "pdf-" or something. This would be similar to the Material Design package for Angular where every class has a "mat-" prefix.

**Describe alternatives you've considered**
Well currently you have to rename your class to get a workaround...

Nice library btw!

**Comment by stephanrauh** [comment]
Dunno. I understand your pain. But I'm afraid this is a breaking change. It'll break many installations of ngx-extended-pdf-viewer. Is there a way to avoid this?

BTW, from a technical point of view, it's not easy to implement you change. pdf.js uses many CSS rules. ngx-extended-pdf-viewer tries to re-use these CSS rules as much as possible. It's not that I've invented my own set of CSS rule; I've inherited it from the upstream project, modifying it here and there.
